src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -103,149 +73,107 @@
-    public static final String INSTRUCTOR_ROLE_ERROR_REASON_NOT_MATCHING =
-            "it does not match the predifined roles";
 
-    /*
-     * =======================================================================
-     * Field: Feedback session start/end times
-     * Start time should be before end time.
-     * Only 1 hour increments allowed.
-     * TODO: allow smaller increments.
-     */
-    public static final String START_TIME_FIELD_NAME = "start time";
-    public static final String END_TIME_FIELD_NAME = "end time";
+    // TODO: allow smaller increments (currently only 1 hour increments allowed)
+    public static final String SESSION_START_TIME_FIELD_NAME = "start time";
+    public static final String SESSION_END_TIME_FIELD_NAME = "end time";
     
-    /*
-     * =======================================================================
-     * Field: Feedback session name
-     */
-    public static final String FEEDBACK_SESSION_NAME = "feedback session";
-    public static final String FEEDBACK_SESSION_NAME_FIELD_NAME = "feedback session name";
-    public static final int FEEDBACK_SESSION_NAME_MAX_LENGTH = 38;
-    public static final String FEEDBACK_SESSION_NAME_ERROR_MESSAGE =
-            "\"%s\" is not acceptable to TEAMMATES as " + FEEDBACK_SESSION_NAME_FIELD_NAME + " because it %s. "
-            + "The value of " + FEEDBACK_SESSION_NAME_FIELD_NAME + " should be no longer than "
-            + FEEDBACK_SESSION_NAME_MAX_LENGTH + " characters. It should not be empty.";
-    
-    /*
-     * =======================================================================
-     * Field: Google ID
-     */
     public static final String GOOGLE_ID_FIELD_NAME = "Google ID";
     public static final int GOOGLE_ID_MAX_LENGTH = 254;
-    public static final String GOOGLE_ID_ERROR_MESSAGE =
-            "\"%s\" is not acceptable to TEAMMATES as a Google ID because it %s. "
-            + "A Google ID must be a valid id already registered with Google. "
-            + "It cannot be longer than " + GOOGLE_ID_MAX_LENGTH + " characters. "
-            + "It cannot be empty.";
     
-    /*
-     * =======================================================================
-     * Field: Gender
-     */
     public static final String GENDER_FIELD_NAME = "gender";
-    public static final List<String> GENDER_ACCEPTED_VALUES = Arrays.asList(Const.GenderTypes.MALE,
-                                                                            Const.GenderTypes.FEMALE,
-                                                                            Const.GenderTypes.OTHER);
+    public static final List<String> GENDER_ACCEPTED_VALUES =
+            Arrays.asList(Const.GenderTypes.MALE, Const.GenderTypes.FEMALE, Const.GenderTypes.OTHER);
+    
+    public static final String GIVER_TYPE_NAME = "feedback giver.";
+    public static final String RECIPIENT_TYPE_NAME = "feedback recipient.";
+    public static final String VIEWER_TYPE_NAME = "feedback viewer.";
```

Comment:
It's because the `PARTICIPANT_TYPE_ERROR_MESSAGE` from which the full error message is constructed does not has the full stop. I'm leaving this error message for a separate issue.

I am dealing mainly with error messages related to `SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE` in this PR. Previously, almost the whole of `FieldValidator` was rather confusing; this was because `SIZE_CAPPED...` constituted a large part of `FieldValidator` and was itself rather confusing. My hope is that after this PR, `FieldValidator` looks like something with many separate mistakes (to be fixed separately), and not one big mistake.